src/node_process.h

```diff
@@ -83,7 +89,7 @@ class BindingData : public SnapshotableObject {
  private:
   static constexpr size_t kBufferSize =
       std::max(sizeof(uint64_t), sizeof(uint32_t) * 3);
-  v8::Global<v8::ArrayBuffer> array_buffer_;
+  bool HrTimeBufferValid();
```

Comment:
Can this be split into a different PR/commit? It would be easier to read to split the `env` -> `realm`/`isolate_data` changes into another commit too, it's a bit difficult to hunt for the important changes from all the refactorings ;)

Reply:
Thanks for the suggestion, I've split the PR into smaller commits.